CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ if(WEVERYTHING)
         -Wno-double-promotion
         -Wno-float-equal
         -Wno-padded
+        -Wno-return-std-move-in-c++11
```

Comment:
@chfast is this ok or should I disable it only for clang somehow?

Reply:
You disabled it only for `-Weverything`. If that's intentional, then we are fine as `-Weverything` only make sense for Clang. I'm assuming `-Wreturn-std-move-in-c++11` is not enabled in default build.